src/Adapters/Laravel/Console/Commands/PanDeleteCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pan\Adapters\Laravel\Console\Commands;
+
+use Illuminate\Console\Command;
+use Pan\Contracts\AnalyticsRepository;
+
+class PanDeleteCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'pan:delete {--id= : The ID of the analytic to delete}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Delete analytics by ID.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(AnalyticsRepository $repository): void
+    {
+        $id = $this->option('id');
```

Comment:
As per @nunomaduro  `pan:delete {id}`, I think we need an argument here at the place of option.
```suggestion
        $id = $this->argument('id');
```

Reply:
Done. Thanks @MrPunyapal .